app/controllers/new-signup.js

```
@@ -0,0 +1,199 @@
+      throw new Error(SIGNUP_ERROR_MESSAGES.invalidInput);
+    }
+    try {
+      const sanitizedFirstname = firstname.toLowerCase();
+      const sanitizedLastname = lastname.toLowerCase();
+
+      const response = await apiRequest(
+        GENERATE_USERNAME_LINK(sanitizedFirstname, sanitizedLastname),
+      );
+
+      const user = await response.json();
+      if (user && user.username) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)